src/components/howTos.stories.mdx

```diff
@@ -0,0 +1,36 @@
+<Meta title="How-tos" />
+
+# How-to guides
+
+There are some particular use cases in this page. You can refer to the usage and some examples below.
```

Comment:
How to guide can have more examples more generally and not only particular right? So, should we say:
"Please check use-cases below for usage examples."

Reply:
emm, I can remove the first sentence.